### PR TITLE
Donator Item - Ready For Review

### PR DESCRIPTION
**Pull request by LetterJay**
DNM this as of yet. Still getting info I needed.

### CODE REVIEW

File: modular_citadel/code/modules/client/loadout/__donator.dm [hunk 1]
```diff
@@ -458,3 +458,34 @@ datum/gear/darksabresheath
...
+
+/datum/gear/mikubikini
+	name = "starlight singer bikini"
+	category = SLOT_W_UNIFORM
+	path = /obj/item/clothing/under/mikubikini
+	ckeywhitelist = list("grandvegeta")
+
+/datum/gear/mikujacket
+	name = "starlight singer jacket"
+	category = SLOT_W_UNIFORM
```
Comment (Ghommie) [inline modular_citadel/code/modules/client/loadout/__donator.dm]:
```suggestion
	category = SLOT_WEAR_SUIT
```

File: modular_citadel/code/modules/client/loadout/__donator.dm [hunk 2]
```diff
@@ -458,3 +458,34 @@ datum/gear/darksabresheath
...
+
+/datum/gear/mikuhair
+	name = "starlight singer hair"
+	category = SLOT_W_UNIFORM
+	path = /obj/item/clothing/head/mikuhair
+	ckeywhitelist = list("grandvegeta")
+
+/datum/gear/mikugloves
+	name = "starlight singer gloves"
+	category = SLOT_W_UNIFORM
```
Comment (Ghommie) [inline modular_citadel/code/modules/client/loadout/__donator.dm]:
```suggestion
	category = SLOT_GLOVES
```

File: modular_citadel/code/modules/client/loadout/__donator.dm [hunk 3]
```diff
@@ -458,3 +458,34 @@ datum/gear/darksabresheath
...
+	path = /obj/item/clothing/gloves/mikugloves
+	ckeywhitelist = list("grandvegeta")
+
+/datum/gear/mikuleggings
+	name = "starlight singer leggings"
+	category = SLOT_W_UNIFORM
```
Comment (Ghommie) [inline modular_citadel/code/modules/client/loadout/__donator.dm]:
```suggestion
	category = SLOT_SHOES
```

File: modular_citadel/code/modules/custom_loadout/custom_items.dm [hunk 4]
```diff
@@ -510,3 +510,48 @@
...
+	mutantrace_variation = NO_MUTANTRACE_VARIATION
+
+/obj/item/clothing/head/mikuhair
+	name = "starlight singer hair"
+	desc = " "
+	icon_state = "mikuhair"
+	item_state = "mikuhair"
+	icon = 'icons/obj/custom.dmi'
+	alternate_worn_icon = 'icons/mob/custom_w.dmi'
+	mutantrace_variation = NO_MUTANTRACE_VARIATION
```
Comment (Ghommie) [inline modular_citadel/code/modules/custom_loadout/custom_items.dm:539]: Add in a `flags_inv = HIDEHAIR` line or the wearer's haircut will probably be also shown...
Not that this wig doesn't stick out like a sore thumb already.

**Review by Ghommie — COMMENTED** [review]
ok...

File: modular_citadel/code/modules/custom_loadout/custom_items.dm (hunk 4 above)
Comment (Ghommie) [reply modular_citadel/code/modules/custom_loadout/custom_items.dm:539]: flag**s**_inv, not flag_inv.

**Review by deathride58 — APPROVED** [review]
Hello Everyone, I’m Scott, President of Domino’s Pizza! Have you heard of Hatsune Miku? Today I'd like to announce a new collaborative project featuring Hatsune Miku: Domino's App featuring Hatsune Miku! Hatsune Miku exists in a software called Vocaloid. Vocaloid enables you to produce songs. A character named Hatsune Miku sings the songs you create. A great feature is you can create songs as you like. I knew our talented Domino's Pizza crew could work together and create great Vocaloid songs. Bokolo P, Esshi, Chokyoshi, Futitsukeshi, everyone! Amazing Vocaloid songs have been created with the fantastic imagination of the crews from all over Japan! The challenge was successfully carried out; and this new collaborative app was produced. ＤＯＭＩＮＯＳ ＰＩＺＡＡᵃ ᵃ ᵃ Based on Miku's image, the Domino's app changes its appearance a lot of music and illustrations produced by Domino's crew are here. From the menu to the order just like Miku. Once your pizza is delivered, have some fun with Miku! it comes with a social camera function and you can take various poses, pictures of Miku. Very cool! And last, but not least, the live performance! Start the pizza stage live and point the camera towards the pizza box and the pizza box will turn into a live dancing venue. A live performance of Luv4Night produced by Domino's crew! Here we go!

**Comment by Linzolle** [comment]
bhijn i fucking hate you

**Comment by Ghommie** [comment]
Bhijn, pls.

**Pull request merged**